PIL/aliases.py

```diff
@@ -0,0 +1,12 @@
+from typing import Tuple
+
+# Type aliases; names subject to change
+LURD = Tuple[int, int, int, int]  # left, up(per), right, down = x0, y0, x1, y1
+XY = Tuple[int, int]
+Coord = XY
+Size = XY  # NOTE: All XY aliases will be interchangeable
+Matrix4 = Tuple[float, float, float, float]
+Matrix12 = Tuple[float, float, float, float, float, float, float, float, float, float, float, float]
+Mode = str
+Extrema = Union[Tuple[Any, Any], Tuple[Tuple[Any, Any], ...]]  # Any -> float?
```

Comment:
As per above, likely Extrema can be simply of floats?

As above, would this cover it?
`Union[float, Tuple[int], Tuple[int, int], Tuple[int, int, int], Tuple[int, int, int, int], Text]` ?

Is Text a special case for `new` ?